api/envoy/data/extensions/filter/udp/dns_filter/v2alpha/inline_dns_table.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package envoy.data.extensions.filter.udp.dns_filter.v2alpha;
```

Comment:
I might consider putting this in an `envoy.data.dns` namespace or similar? I could see this being used elsewhere beyond this filter later?

Reply:
I can move it